nf_core/pipeline-template/assets/methods_description_template.yml

```diff
@@ -0,0 +1,25 @@
+id: "{{ name_noslash }}-methods-description"
+description: "Suggested methods description text of pipeline usage."
+section_name: "{{ name }} Methods Description"
+section_href: "https://github.com/{{ name }}"
+plot_type: "html"
+## TODO nf-core: Update the HTML below to your prefered methods description, e.g. add publication citation for this pipeline
+## You inject any metadata in the Nextflow '${workflow}' object
+data: |
+  <h4>Methods</h4>
+  <p>Data was processed using {{ name }} v${workflow.manifest.version} (DOI: <a href="https://doi.org/${workflow.manifest.doi}">${workflow.manifest.doi}</a>) of the nf-core collection of workflows (<a href="https://doi.org/10.1038/s41587-020-0439-x">Ewels <em>et al.</em>, 2020</a>).</p>
```

Comment:
DOI is missing in the example. Maybe we should add some logic here to only print it if it exists?

Reply:
I'm not sure if I can get this to work, as it will be nesting variables within variables when we add htem using `.make(meta)` with the template engine, but I'll give it a try and see what happens...